statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -759,35 +748,36 @@ def update(self, params, transformed=True, set_params=True):
 
         if not transformed:
             params = self.transform_params(params)
-        if set_params:
-            self.params = params
-
-        if self.updater is not None:
-            self.updater(self, params)
 
         return params
 
     @classmethod
     def from_formula(cls, formula, data, subset=None):
         """
-        Not implemented for State space models
+        Not implemented for state space models
         """
         raise NotImplementedError
 
 
-class MLEResults(FilterResults, tsbase.TimeSeriesModelResults):
+class MLEResults(tsbase.TimeSeriesModelResults):
     r"""
     Class to hold results from fitting a state space model.
 
     Parameters
     ----------
-    model : Model instance
+    model : MLEModel instance
         The fitted model instance
+    params : array
+        Fitted parameters
+    ssm_result : KalmanFilter instance
+        The underlying state space model and Kalman filter output
```

Comment:
This should get a bit more explanation, here or in a Notes section.

one question: why is it `ssm_results` and not `ssm`? AFAICS, because it's the filtered state, not a "raw" statespace model?